Source/Scene/ModelAnimation.js

```diff
@@ -230,4 +240,25 @@ Object.defineProperties(ModelAnimation.prototype, {
     },
   },
 });
+/**
+ * A function used to compute the local animation time for a ModelAnimation.
+ * @callback ModelAnimation.AnimationTime
+ *
+ * @param {Number} duration The animation's original duration in seconds.
+ * @param {Number} seconds Scene time seconds since the animation started.
+ * @returns {Number} The fractional part of the return value gives
+ *          the fraction of the animation that has been completed.
+ *          If the animation's loop option is set to REPEAT the
+ *          integer part of the result is ignored; if its set to
+ *          MIRROR_REPEAT the animation runs backwards when the
+ *          integer part is odd, and forwards when its even.
+ *          Otherwise the animation will be pinned to its start
+ *          for negative values, and the end for values greater
+ *          than 1.0
+ *
+ * @example
+ * function animationTime(duration) {
```

Comment:
Suppose I have several copies of a model, and I want the default animationTime behavior, except that they should all be out of phase with each other. I can just define something like
```
function getAnimationTimeFn(offset) {
  return function(duration, seconds) { return seconds / duration + offset }
}
```

In the same way, it would solve the problem that #10043 is trying to solve (I commented there that the current proposal seems broken).

Reply:
That makes sense, thanks for explaining. I think we should add an example like that to the documentation, as a second `@example` underneath the one already here